spec/rubocop/cop/style/collection_methods_spec.rb

```diff
@@ -42,8 +42,8 @@
     end
 
     it 'auto-corrects to preferred method' do
-      new_source = autocorrect_source('some.collect(&:test)')
```

Comment:
This was a bug from before where it tested `collect: 'map'` for every preferred method, instead of testing every `method: 'preferred_method'` autocorrect